default_validator/src/lib.rs

```diff
@@ -1,38 +1,44 @@
 //! Iroha default validator.
+
```

Comment:
This file is a little confusing. A while ago I fully expected to find the entire validator here as a reference implementation. 

Why did you decide to make this a separate crate, but also not to provide the actual logic inside of it? Wouldn't it make more sense to inline at least the top level functions instead of just having one line per function?

Reply:
Closed in favour of #3765